oqtopus/gui/parameters_groupbox.py

```diff
@@ -57,7 +87,10 @@ def setParameters(self, parameters: list[ParameterDefinition]):
     def parameters_values(self):
         values = {}
         for parameter in self.parameters:
-            values[parameter.name] = self.parameter_widgets[parameter.name].value()
+            if self.parameter_widgets[parameter.name].valueSet():
+                values[parameter.name] = self.parameter_widgets[parameter.name].value()
+            # else:
+            #     values[parameter.name] = parameter.default
```

Comment:
@3nids if I set the parameter.default for other param than the SRID I get this error. I guess even `parameter.default` should be cast according to parameter.type?

```
Can't install the module:
Error applying changelog /tmp/oqtopus/wastewater-34/datamodel/changelogs/2025.0.1/03_tww_db_dss.sql: SQL preparation failed for the following code: COMMENT ON COLUMN tww_od.measure.intervention_demand IS 'Short description of need of action / Kurzbeschreibung des Handlungsbedarfs / Description courte du besoin d''intervention' cannot adapt type 'Literal' using placeholder '%t' (format: TEXT)
```